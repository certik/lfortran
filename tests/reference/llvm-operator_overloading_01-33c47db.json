{
    "basename": "llvm-operator_overloading_01-33c47db",
    "cmd": "lfortran --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/../integration_tests/operator_overloading_01.f90",
    "infile_hash": "f44f3172b4b5235b41deb5a4d4eaadc61a6b7c51e002a983889cf38f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-operator_overloading_01-33c47db.stdout",
    "stdout_hash": "59b7bf49d9a6374fa600cddd2b03c7fcbc8ccb4253d6348f87a29720",
    "stderr": "llvm-operator_overloading_01-33c47db.stderr",
    "stderr_hash": "05abbb654410de494e99375db5211f1338b5b975110521ec045a5d04",
    "returncode": 1
}