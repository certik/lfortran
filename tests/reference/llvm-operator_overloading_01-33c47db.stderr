style suggestion: Use 'end if' instead of 'endif'
  --> tests/../integration_tests/operator_overloading_01.f90:28:9
   |
28 |         endif
   |         ^^^^^ help: write this as 'end if'

code generation error: asr_to_llvm: module failed verification. Error:
Both operands to ICmp instruction are not of the same type!
  %13 = icmp eq i32 %12, i1 false
Both operands to ICmp instruction are not of the same type!
  %16 = icmp eq i32 %15, i1 false
Both operands to ICmp instruction are not of the same type!
  %19 = icmp eq i32 %18, i1 false
Both operands to ICmp instruction are not of the same type!
  %22 = icmp eq i32 %21, i1 false

